modules/nw-multus-add-pod.adoc

```diff
@@ -26,7 +28,7 @@ $ oc edit pod <name>
 
 . In the Pod resource definition, add the `k8s.v1.cni.cncf.io/networks`
 parameter to the Pod `metadata` mapping. The `k8s.v1.cni.cncf.io/networks`
-accepts a comma separated string of one or more `AdditionalNetworkDefinition`
+accepts a comma separated string of one or more NetworkAttachmentDefinition
```

Comment:
Do we need mark-up? `NetworkAttachmentDefinition`

Reply:
I'm not sure if we have a specific style for this; I'd prefer to only use a code block to reference values and commands. This is a CR _Kind_, so I think of it as the name of the CR thing, since all CRs of `Kind: NetworkAttachmentDefinition` describe an additional network. Although it's _also_ a parameter value, but not in this context. So the use is bifurcated and thus I'm not sure which is appropriate universally.